apps/omg_watcher/test/fixtures.exs

```diff
@@ -168,34 +166,48 @@ defmodule OMG.Watcher.Fixtures do
 
       (started_apps ++ started_security_watcher ++ started_watcher_api)
       |> Enum.reverse()
-      |> Enum.map(fn app -> :ok = Application.stop(app) end)
+      |> Enum.map(fn app ->
+        :ok = Application.stop(app)
+      end)
+
+      Process.sleep(5_000)
     end)
   end
 
   deffixture test_server do
-    {:ok, server_id, port} = Server.run()
-    env = FakeServer.Env.new(port)
-
-    EnvAgent.save_env(server_id, env)
+    server_id = random_atom()
+    {:ok, pid} = FakeServer.start(server_id)
 
     real_addr = Application.fetch_env!(:omg_watcher, :child_chain_url)
     old_client_env = Application.fetch_env!(:omg_watcher, :child_chain_url)
-    fake_addr = "http://#{env.ip}:#{env.port}"
+    {:ok, port} = FakeServer.port(server_id)
+    fake_addr = "http://localhost:#{port}"
 
     on_exit(fn ->
       Application.put_env(:omg_watcher, :child_chain_url, old_client_env)
 
-      Server.stop(server_id)
-      EnvAgent.delete_env(server_id)
+      FakeServer.stop(server_id)
     end)
 
     %{
       real_addr: real_addr,
       fake_addr: fake_addr,
-      server_id: server_id
+      server_id: server_id,
+      server_pid: pid
     }
   end
 
+  defp random_atom() do
+    chars = String.split("ABCDEFGHIJKLMNOPQRSTUVWXYZ", "")
+
+    1..10
+    |> Enum.reduce([], fn _i, acc ->
+      [Enum.random(chars) | acc]
+    end)
+    |> Enum.join("")
+    |> String.to_atom()
+  end
```

Comment:
what's going on here?

Reply:
@InoMurko I removed it. I hardcoded server_id for fake_server instead. Now all tests use fake_server with the same server_id